### PR TITLE
Time travel cleanup

### PR DESCRIPTION
**Pull request by Frawstcrabs**
I am not the best at managing pull requests.

### CODE REVIEW

File: dapp/src/components/TimeTravelWidget.tsx [hunk 1]
```diff
@@ -117,6 +142,61 @@ export default function TimeTravelWidget({
...
+      if (input === null) {
+        return;
+      }
+      const newBlock = parseInt(input || "");
+      if (
+        isNaN(newBlock) ||
+        newBlock < parseInt(timeTravelRange.min.number) ||
+        newBlock > parseInt(timeTravelRange.max.number)
+      ) {
+        alert(`Invalid page number: ${input}`);
```
Comment (opatdchan) [inline dapp/src/components/TimeTravelWidget.tsx:191]: Just noticed I left a copypaste here, should be block number.

**Review by opatdchan — APPROVED** [review]
Looks good so far, if you give me the ok I'm merging.
Thank you, and sorry for breaking master so often. These are unstable times.

**Comment by Frawstcrabs** [comment]
I've been using my local client for a while and I haven't gotten any issues with the time travel so far, so it should be good to merge.

**Comment by opatdchan** [comment]
This is so much better it's unreal

**Pull request merged**